.github/workflows/gsy-e_ci.yaml

```diff
@@ -9,15 +9,33 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+
+    - uses: 8BitJonny/gh-get-current-pr@1.3.0
```

Comment:
What a username :-D

Reply:
I did not noticed this as I just took Marcelos suggestion and added it on this PR. It is a cool one indeed XD .